samples/crypto/psa_tls/overlays/oberon-legacy.conf

```diff
@@ -0,0 +1,2 @@
+CONFIG_CC3XX_BACKEND=n
```

Comment:
Unsure about negative prompt will exist for CONFIG_CC3XX_BAKEND. Please remove this

Reply:
Removed, internal run still green and verified cc3xx was not turned on.